test/modules/MockModule.ts

```diff
@@ -43,3 +50,64 @@ export function registerMockModule(): MockModule {
     }
     return module;
 }
+
+export class MockModuleWithCryptoSetupExtension extends RuntimeModule {
```

Comment:
Again, please document new `exported` symbols, even in test code.

Reply:
Please do remember do document the `export`ed symbols in this file. A quick overview of their purpose would be sufficient.